setup.py

```diff
@@ -19,10 +19,15 @@
     package_data={'': ['LICENSE']},
     packages=find_packages(),
     install_requires=install_requires,
+    scripts = ['parsl/executors/high_throughput/process_worker_pool.py',
+               'parsl/executors/extreme_scale/mpi_worker_pool.py'],
     extras_require = {
         'db_logging' : ['CMRESHandler', 'psutil', 'sqlalchemy'],
         'aws' : ['boto3'],
```

Comment:
we should add instructions on how to install extras to docs

Reply:
Done. please check the updated parsl/docs/quickstart.rst